Withings/Models/BloodPressure.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Withings.API.Portable.Models
+{
+    class BloodPressure
```

Comment:
can we put a comment here (and in each of the models) that links out to where this model comes from in the withings API? That'll help me double check the work! 👍